src/__tests__/user.spec.js

```diff
@@ -1,18 +1,73 @@
 const request = require("supertest");
 const app = require("../../index");
 const db = require("../db/testDb");
+const { createSignature } = require("@leapchain/dleap");
+const getOrCreateUser = require("../utils/getOrCreateUser");
+const {
+  GOVERNOR_REQUEST_FEE,
+  TREASURY_ACCOUNT_NUMBER,
+  MemoType,
+} = require("../constants");
 
-beforeAll(async () => await db.connect());
+beforeAll(async () => {
+  await db.connect();
+  secretKey =
+    "705a3b36b44c2e95eebba870dc6fe5280c5c72783e2fe9449ebb73556faec57e";
+  publicKey =
+    "a5f0766103d33981ca0e90179fbd7ece2c9e76e0ea5d99ebb86cb80ff3c99ab9";
+});
```

Comment:
dont keep that in git

Reply:
i was actually using the account pairs used https://github.com/LeapChain/DLEAP/blob/main/javascript/test/sign.test.ts
changed it to 00000 to make it obvious